packages/11ty/_plugins/figures/figure-factory.js

```diff
@@ -0,0 +1,95 @@
+const AnnotationFactory = require('./annotation-factory')
+const chalkFactory = require('~lib/chalk')
+const Tiler = require('./iiif/tiler')
+const transform = require('./transform')
+const Manifest = require('./iiif/manifest')
+const ManifestWriter = require('./iiif/manifest/writer')
+const logger = chalkFactory('Figure Processing')
+const { getPrintImage, isCanvas, isImageService } = require('./helpers')
```

Comment:
Sort `require` expressions before other `const` declaration expressions.
```suggestion
const AnnotationFactory = require('./annotation-factory')
const chalkFactory = require('~lib/chalk')
const Tiler = require('./iiif/tiler')
const transform = require('./transform')
const Manifest = require('./iiif/manifest')
const ManifestWriter = require('./iiif/manifest/writer')
const { getPrintImage, isCanvas, isImageService } = require('./helpers')

const logger = chalkFactory('Figure Processing')
```